core/src/test/java/org/apache/carbondata/core/datastore/filesystem/LocalCarbonFileTest.java

```diff
@@ -44,8 +45,8 @@
 
     @BeforeClass
     static public void setUp() {
-        file = new File("Test.carbondata");
-        dir = new File("Testdir.carbondata");
+        file = new File("TestLocalCarbonFile");
```

Comment:
FYI:
Previous file name is the same as that in other testcase and this will cause failure in windows env due to file corruption